astraconnector_operator.yaml

```diff
@@ -1,496 +0,0 @@
-apiVersion: v1
```

Comment:
I added this thinking this was the original issue. 
This file was moved to details/operator-sdk/astraconnector_operator.yaml - so we don't actually need this one here.